tests/conftest.py

```diff
@@ -454,3 +455,12 @@ def cli_runner():
 def cli_fs_runner(cli_runner):
     with cli_runner.isolated_filesystem():
         yield cli_runner
+
+
+@pytest.fixture(scope="function")
+def isolated_fs(fs):
+    # isolate fs but include CA bundle for https validation
+    fs.add_real_directory(os.path.dirname(extract_zipped_paths(DEFAULT_CA_BUNDLE_PATH)))
+    # add cassetes dir
+    cassetes_dir = join(dirname(realpath(__file__)), "cassettes")
```

Comment:
typo: casset**t**es

Reply:
? seems to be ok here

Reply:
I meant the variable name (and matching comment)

Reply:
fair point, didn't notice it